# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1349248608 14400
#      Wed Oct 03 03:16:48 2012 -0400
# Node ID 85e3ebc11da4bf187ee6599e8f4e641da68cfb9f
# Parent  c7d7d43f26bba736d06316dcc17d27b0fb9ff663
* variables.cc (bind_internal_variable): Tag with GCC_ATTR_DEPRECATED.

diff --git a/libinterp/interpfcn/variables.h b/libinterp/interpfcn/variables.h
--- a/libinterp/interpfcn/variables.h
+++ b/libinterp/interpfcn/variables.h
@@ -131,17 +131,18 @@ set_internal_variable (int& var, const o
 
 extern OCTINTERP_API std::string builtin_string_variable (const std::string&);
 extern OCTINTERP_API int builtin_real_scalar_variable (const std::string&, double&);
 extern OCTINTERP_API octave_value builtin_any_variable (const std::string&);
 
 extern OCTINTERP_API void bind_ans (const octave_value& val, bool print);
 
 extern OCTINTERP_API void
-bind_internal_variable (const std::string& fname, const octave_value& val);
+bind_internal_variable (const std::string& fname,
+                        const octave_value& val) GCC_ATTR_DEPRECATED;
 
 extern OCTINTERP_API void mlock (void);
 extern OCTINTERP_API void munlock (const std::string&);
 extern OCTINTERP_API bool mislocked (const std::string&);
 
 extern OCTINTERP_API void clear_function (const std::string& nm);
 extern OCTINTERP_API void clear_variable (const std::string& nm);
 extern OCTINTERP_API void clear_symbol (const std::string& nm);
